frontend/components/common/Login.tsx

```diff
@@ -1,28 +1,50 @@
-import React from "react";
+import React , {useState, useEffect}from "react";
 import { ConnectButton } from "web3uikit";
 
 function Login() {
+  const [isSmallScreen, setIsSmallScreen] = useState(false);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setIsSmallScreen(window.innerWidth <= 600);

```

Comment:
how will you set different design for mobile,bigger phones, tablets,laptop and desktop. This basically divided everything into 2 huge chunks, i think this should work just fine tho.

@J0SAL do we need to maintain a very high standard of code ? this will work but can be done better for better customer experience.

Reply:
> how will you set different design for mobile,bigger phones, tablets,laptop and desktop. This basically divided everything into 2 huge chunks, i think this should work just fine tho.
> 
> @J0SAL do we need to maintain a very high standard of code ? this will work but can be done better for better customer experience.

ok I will update it.

Reply:
> how will you set different design for mobile,bigger phones, tablets,laptop and desktop. This basically divided everything into 2 huge chunks, i think this should work just fine tho.
> 
> @J0SAL do we need to maintain a very high standard of code? this will work but can be done better for better customer experience.

I believe some good code practices including a decent level of code modularity should be enough